### PR TITLE
feature/CORE-538/CORE-631/cypress-eslint-ts-improvements

### PR DESCRIPTION
**Pull request by bulatgab**

### CODE REVIEW

File: cypress/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "cypress/globals": true
+  },
+  "plugins": [
+    "cypress"
+  ],
+  "extends": [
+    "plugin:cypress/recommended"
+  ],
+  "rules" : {
+    "import/no-extraneous-dependencies": "off",
```
Comment (bulatgab) [inline cypress/.eslintrc.json:12]: this lets us put cypress libraries (like cypress-drag-drop) in devDependencies and import them

File: cypress/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "@typescript-eslint/no-namespace": "off"
```
Comment (bulatgab) [inline cypress/.eslintrc.json:13]: this lets us do
```
declare global {
  namespace Cypress {
...
```

File: package.json [hunk 3]
```diff
@@ -125,6 +125,7 @@
     "eslint-config-airbnb": "^18.2.1",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-config-airbnb-typescript": "^12.3.1",
+    "eslint-plugin-cypress": "^2.13.3",
```
Comment (bulatgab) [inline package.json:128]: better checking for our cypress code

**Review by object-required — APPROVED** [review]
Looks great 🔥

File: package.json (hunk 3 above)
Comment (safydy-r) [reply package.json:128]: Love this, thanks @bulatgab

**Pull request merged**